stock_picking_validate_jobcan_wf/models/stock_picking.py

```diff
@@ -96,11 +96,6 @@ def _check_jobcan_wf_status(self):
     def notify_user(self, message):
         self.ensure_one()
         if self.user_id:
```

Comment:
We should remove this condition as we are no longer relying on this field.

```suggestion
```

Reply:
We should also adjust the method name. Change it to `jobcan_validate_error_message_post()` or similar perhaps?